.pre-commit-config.yaml

```diff
@@ -63,3 +62,7 @@ repos:
     hooks:
       - id: pip-audit
         args: ["--require-hashes", "--requirement", "requirements/local.txt"]
+  - repo: https://github.com/antonbabenko/pre-commit-terraform
+    rev: "v1.77.1"
+    hooks:
+      - id: terraform_fmt
```

Comment:
This requires `terraform` to be installed and it is not the case either in our standard Docker images or in some developers systems.

Reply:
For the moment I have removed the terraform hook from precommit config, so as not to block the merge of this PR and discuss its addition in a special PR in which to better evaluate it.